libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKLoginHostListViewController.m

```diff
@@ -175,13 +161,29 @@ - (void)viewWillAppear:(BOOL)animated {
     // We need to make sure the table is refreshed
     // and the size updated when we appear because
     // a new host could have been added by the user.
-    
+    [self setupBrandingForNavBar];
     [self.tableView reloadData];
     [self resizeContentForPopover];
     // style navigiation bar
     [super viewWillAppear:animated];
 }
 
+- (void)setupBrandingForNavBar {
+    [self.navigationController.navigationBar setTranslucent:false];
```

Comment:
Why are we setting translucent to false?

Reply:
@harry528tt I agree with @bhariharan.  Why cant this be done in your own subclass?

Reply:
@bhariharan @trooper2013 I can move the setTranslucent to my own class. But for the remaining part, yes it's also happening on your sample app.